web.config

```diff
@@ -117,14 +117,14 @@
         -->
         <!-- <iisnode watchedFiles="web.config;*.js"/> -->
         <staticContent>
+            <!-- See https://docs.microsoft.com/en-us/iis/configuration/system.webserver/staticcontent/clientcache for more info -->
+            <clientCache cacheControlMode="UseMaxAge" cacheControlMaxAge="365.00:00:00" cacheControlCustom="immutable" />
             <remove fileExtension=".css"/>
             <mimeMap fileExtension=".css" mimeType="text/css; charset=utf-8"/>
             <remove fileExtension=".html" />
             <mimeMap fileExtension=".html" mimeType="text/html; charset=utf-8" />
             <remove fileExtension=".js"/>
             <mimeMap fileExtension=".js" mimeType="text/javascript; charset=utf-8"/>
-            <remove fileExtension=".json"/>
-            <mimeMap fileExtension=".json" mimeType="application/json; charset=utf-8"/>
```

Comment:
@molant You removed this by mistake, or just because we don't have `.json` files

Reply:
We don't have .json files AFAIK

Reply:
@molant The `ttf` ones can be removed. Don't think we are using them anywhere.